### PR TITLE
Release trv10 2.0.1

### PR DESCRIPTION
**Pull request by mofahsan**
- update changelog
- add enery_types field in on_demand examples

### CODE REVIEW

File: api/components/app.js [hunk 1]
```diff
@@ -297,9 +297,9 @@ async function getSwaggerYaml(example_set, outputPath) {
     const { paths } = baseYAML;
     let hasTrueResult = false; // Flag variable
     let schemaMap = {};
+    await buildAttribiutes();
```
Comment (anuj-ondc) [inline api/components/app.js]: Please revert this & add it to default condition.

File: api/components/build-attributes.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ const fs = require("fs");
 async function buildAttribiutes() {
   let attributes = {};
   const workSheetsFromBuffer = xlsx.parse(`./on-demand_attributes_updated.xlsx`);
+  console.log(workSheetsFromBuffer)
```
Comment (anuj-ondc) [inline api/components/build-attributes.js:8]: Please remove console.

File: api/components/app.js (hunk 1 above)
Comment (mofahsan) [reply api/components/app.js]: Resolved

File: api/components/build-attributes.js (hunk 2 above)
Comment (mofahsan) [reply api/components/build-attributes.js:8]: Resolved

**Pull request merged**